README.md

```diff
@@ -9,22 +9,26 @@
 [![Build Status](https://circleci.com/gh/pyodide/pyodide.png)](https://circleci.com/gh/pyodide/pyodide)
 [![Documentation Status](https://readthedocs.org/projects/pyodide/badge/?version=stable)](https://pyodide.readthedocs.io/?badge=stable)
 
-Python with the scientific stack, compiled to WebAssembly.
+Pyodide is a port of the Python 3.9 interpreter and many common Python packages
+to the browser.
 
 ## What is Pyodide?
 
-Pyodide may be used in any context where you want to run Python inside a web
-browser.
+Pyodide is a port of Python to WebAssembly/[Emscripten](https://emscripten.org/)
+for use in the browser or in node.
 
-Pyodide brings the Python 3.9 runtime to the browser via WebAssembly thanks to
-[Emscripten](https://emscripten.org/).
-It builds the Python scientific stack including NumPy, Pandas, Matplotlib, SciPy, and
-scikit-learn. The [packages directory](packages) lists over 75 packages which
-are currently available. In addition, it's possible to install pure Python wheels
+We also port over 80 common Python extensions written in C, C++, and fortran.
+Supported packages range from pyyaml and lxml, which use C extension to do fast
+yaml and xml processing, to standard scientific computing packages like numpy,
+pandas, scipy, and matplotlib. It is also possible to use pure Python wheels
 from PyPI.
```

Comment:
In our current description really focuses on those 80 packages, which raises the question of why so few. Installing from PyPi is added as an afterthought.

To be more general, maybe,
```suggestion
Pyodide allows to install and run Python packages in the browser with [micropip](https://pyodide.org/en/stable/usage/api/micropip-api.html),
 - pure Python packages can be installed as wheels from PyPi
 - many [common packages](https://github.com/pyodide/pyodide/tree/main/packages) with C extensions are built as part of Pyodide. These range from general-purpose packages such as regex, pyyaml, lxml to 
 scientific Python packages including numpy, pandas, scipy, matplotlib, and scikit-learn.
```

It's not fully accurate, because we also build pure Python packages, but it would illustrate the direction where we are going.